packages/client/e2e/limit-checker.spec.ts

```diff
@@ -78,8 +83,9 @@ test.describe("Limit Checker", () => {
     })
   })
 
-  test("Trade is blocked if notional is above limit", async ({}, testInfo) => {
-    test.skip(testInfo.project.name !== OPENFIN_PROJECT_NAME, "Openfin Only")
+ // eslint-disable-next-line no-empty-pattern
+  test("Trade is blocked if notional is above limit", async ({}, workerInfo) => {
+    if (!isOpenFin(workerInfo)) test.skip()
```

Comment:
just noticed this ... what we had before - [conditional skip](https://playwright.dev/docs/test-annotations#conditionally-skip-a-test) - is better (conventions again ..)

Reply:
I moved all `test.skip` occurrences to the former syntax 👍